### PR TITLE
some light content edits

### PR DESCRIPTION
**Pull request by akristen**

### CODE REVIEW

File: src/content/docs/tutorial-app-performance/instrument-your-site.mdx [hunk 1]
```diff
@@ -103,9 +104,9 @@ There are two basic ways to install the browser monitoring agent:
     </TabsPages>
   </Tabs>
 
-## Make data-informed changes [#data]
+## Analyze site performance [#data]
 
-Once you've instrumented your site, you can overview your site's performance with different types of data.
+Once you've instrumented your site, you can overview your site's performance with different types of data. Measuring data
```
Comment (jeff-colucci) [inline src/content/docs/tutorial-app-performance/instrument-your-site.mdx:109]: Seems to just be a hanging Measuring data here?

File: src/content/docs/tutorial-app-performance/performance-benchmarks.mdx [hunk 2]
```diff
@@ -295,13 +295,18 @@ Defining and managing your service levels following the steps below provides the
...
 
-Congratulations! You've completed our journey on how to use New Relic to improve your site's performance! Ready to learn more about using New Relic like a pro? See some of our other tutorials:
+Congratulations! You've completed our journey on how to use New Relic to improve your site's performance! In this tutorial, you learned a few things about improving your site performance. You learned how to:
 
-<DocTiles>
-    <DocTile title='My app is slow' number='1' path="/docs/tutorial-app-slow/root-causes/">Is your app slow? Check out our tutorial to diagnose and resolve app slowness.</DocTile>
-    <DocTile title='Prepare for peak demand events' number='2' path="/docs/tutorial-peak-demand/get-started/">Prepare your teams for a game day so your app stays live when demand peaks.</DocTile>    
-    <DocTile title='Create alerts' number='3' path="/docs/tutorial-create-alerts"> Stay in the loop about your system's health by creating high quality alerts.</DocTile>
-</DocTiles>
+*  Unlock data that can give you insight into how your site currently performs by instrumenting your site.
```
Comment (jeff-colucci) [inline src/content/docs/tutorial-app-performance/performance-benchmarks.mdx:302]: Extra space between the bullet and Unlock I think

File: src/content/docs/tutorial-app-performance/instrument-your-site.mdx (hunk 1 above)
Comment (akristen) [reply src/content/docs/tutorial-app-performance/instrument-your-site.mdx:109]: ASDJGLASF

Comment (akristen) [reply src/content/docs/tutorial-app-performance/instrument-your-site.mdx:109]: Frodo smiling emoji.

**Review by homelessbirds — APPROVED** [review]

**Review by jeff-colucci — APPROVED** [review]

**Pull request merged**